### PR TITLE
Renames some class members for FootClass and TeamClass.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/game/engine/foot.h [hunk 1]
```diff
@@ -167,12 +169,12 @@ class FootClass : public TechnoClass
     target_t m_SuspendedNavCom;
     target_t m_NavList[NAV_LENGTH];
     GamePtr<TeamClass> m_Team;
-    TeamNumberType m_field_113;
-    FootClass *m_field_114; // Next team member?
+    TeamGroupType m_Group;
+    FootClass *m_Member; // Next linked team member.
     FacingType m_Paths[PATH_LENGTH];
-    MoveType m_PathMove; // The move type the current path was generated with.
+    MoveType m_PathBrk; // The move type the current path was generated with. // Perhaps "PathBreak"?
```
Comment (OmniBlade) [inline src/game/engine/foot.h]: I'd prefer PathBreak, PathBrk seems unneccessarily concise? I'm aware its what the symbols show it was called originally.

File: src/game/engine/radio.cpp [hunk 2]
```diff
@@ -13,17 +13,18 @@
...
+ // A invalid target instance parsed into targets as a default param.
 target_t &RadioClass::Get_LParam()
 {
-    return LParam;
+    static target_t _lparam = 0;
+    return _lparam;
 }
 
-const char *RadioClass::Messages[RADIO_COUNT] = {
+#ifdef CHRONOSHIFT_DEBUG
```
Comment (OmniBlade) [inline src/game/engine/radio.cpp]: I don't think there is any need for the array to be behind the debug macro, if its compiled in but never used, what does it matter?

File: src/game/engine/radio.cpp [hunk 3]
```diff
@@ -181,34 +195,36 @@ RadioMessageType RadioClass::Transmit_Message(RadioMessageType message, RadioCla
...
-        Radio = reinterpret_cast<ObjectClass *>(As_Techno((uintptr_t)(Radio)));
+    if (m_Radio != nullptr) {
+        m_Radio = reinterpret_cast<ObjectClass *>(As_Techno((uintptr_t)(m_Radio)));
+        DEBUG_ASSERT(m_Radio != nullptr);
     }
 
     MissionClass::Decode_Pointers();
 }
 
+#ifdef CHRONOSHIFT_DEBUG
```
Comment (OmniBlade) [inline src/game/engine/radio.cpp]: Why are we gating this behind the debug macro when it was in the release build?

**Review by OmniBlade — CHANGES_REQUESTED** [review]
I'm not sure why you've gated some of the message stuff behind the debug macro, its a bit out of scope for a member renaming PR and could do with more discussion.

File: src/game/engine/team.h [hunk 4]
```diff
@@ -58,16 +58,16 @@ class TeamClass : public AbstractClass
...
 #ifndef CHRONOSHIFT_NO_BITFIELDS
-    BOOL m_Bit1_1 : 1; // 1
-    BOOL m_Bit1_2 : 1; // 2
-    BOOL m_Bit1_4 : 1; // 4
-    BOOL m_ForcedActive : 1; // 8
-    BOOL m_Bit1_16 : 1; // 16
+    BOOL m_ForcedActive : 1; // 1
+    BOOL m_HasBeen : 1; // 2
+    BOOL m_FullStrength : 1; // 4
+    BOOL m_Bit1_8 : 1; // 8
```
Comment (tomsons26) [inline src/game/engine/team.h]: 8 is Understrength

**Review by OmniBlade — APPROVED** [review]

**Pull request merged**